src/MongoDB/Driver/Manager.cpp

```diff
@@ -53,6 +53,7 @@ const StaticString
 	s_MongoDBDriverManager_readpreferencetags("readpreferencetags"),
 	s_MongoDBDriverManager_readPreference("readPreference"),
 	s_MongoDBDriverManager_readPreferenceTags("readPreferenceTags"),
+	s_MongoDBDriverManager_maxStalenessMS("maxStalenessMS"),
```

Comment:
In the interest of consistency, would you like to create a PHPC ticket to ultimately remove the case-insensitive handling of `$uriOptions`? [PHPC-655](https://jira.mongodb.org/browse/PHPC-655) is one related ticket, which we actually have a test for parsing the all-caps variant; however, that test is likely obsolete now that I've dropped PHP streams.